### PR TITLE
docs: community projects

### PR DESCRIPTION
**Pull request by vinckr**
@atreya2011 adding your WIP project, great work thanks!

@sergiocarracedo Thanks for your article on dockertest, very cool!
Let us know if you would be interested in speaking at an Ory Event in the near future 😃

### CODE REVIEW

**Comment by sergiocarracedo** [comment]
Thank so much. My pleasure

File: docs/docs/ecosystem/community.md [hunk 1]
```diff
@@ -61,14 +61,16 @@ This is not an exhaustive list. Want your own content here? Create a PR!
...
+- [WIP Ory Hydra/Kratos Integration in Go](https://github.com/atreya2011/go-kratos-test)
 - [WIP: Ory Kratos React example](https://github.com/realStandal/kratos-react-example)
 - [WIP: Ory Hydra for GCP](https://github.com/someone1/hydra-gcp)
 - [WIP: Ory Hydra/ Ory Oathkeeper Zero Trust Reference "Hello World App"](https://github.com/JasonCubic/oathkeeper_hydra_reverse_proxy)
 
+- [Archived: Ory Hydra Golang login/consent provider using Authboss user/password authentication](https://github.com/nbycomp/login-consent)
+
 ### Articles & Tutorials
 
+- [Integration tests in Golang with dockertest ](https://sergiocarracedo.es/integration-tests-in-golang-with-dockertest/)
```
Comment (aeneasr) [inline docs/docs/ecosystem/community.md:73]:
```suggestion
- [Integration tests in Golang with dockertest](https://sergiocarracedo.es/integration-tests-in-golang-with-dockertest/)
```

**Comment by atreya2011** [comment]
@vinckr Thank you for featuring my WIP project in the official documentation 🙇🏼‍♂️

**Comment by vinckr** [comment]
Sure @atreya2011, it is already helpful to the wider community! 
Feel free to ping/DM me when you feel we can remove the `WIP` tag :-)

**Pull request merged**